### PR TITLE
Use user's id

### PR DESCRIPTION
**Pull request by Slawisch**

### CODE REVIEW

File: frontend/src/app/modules/main/playlist/artist-list/artist-list.component.ts [hunk 1]
```diff
@@ -12,9 +14,13 @@ export class ArtistListComponent implements OnInit {
   userId: number = 1; // Temporary value
   artists!: ArtistReadDTO[];
 
-  constructor(
-    private _reactionService: ReactionService
-  ) { }
+  constructor(private _reactionService: ReactionService, private _authService: AuthService) {
+    this._authService.getAuthStateObservable()
+      .pipe(filter((state) => state !== null))
```
Comment (proskurnicha) [inline frontend/src/app/modules/main/playlist/artist-list/artist-list.component.ts]: You could simplify it
.pipe(filter((state) => state !== null)) -> .pipe(filter((state) => state))

File: frontend/src/app/modules/main/playlist/all/all.component.ts [hunk 2]
```diff
@@ -19,9 +20,14 @@ export class AllComponent {
   userId: number = 1;
   likedSongs: number = 256;
 
-  constructor(private _reactionService: ReactionService) {
+  constructor(private _reactionService: ReactionService, private _authService: AuthService) {
     this.loadPlaylist();
     this.loadAlbums();
+    this._authService.getAuthStateObservable()
```
Comment (proskurnicha) [inline frontend/src/app/modules/main/playlist/all/all.component.ts:24]: the same here

File: frontend/src/app/modules/main/playlist/album-list/album-list.component.ts [hunk 3]
```diff
@@ -13,8 +15,15 @@ export class AlbumListComponent implements OnInit {
   albums!: AlbumForReadDTO[];
 
   constructor(
-    private _reactionService: ReactionService
-  ) { }
+    private _reactionService: ReactionService,
+    private _authService: AuthService
+  ) {
+    this._authService.getAuthStateObservable()
+      .pipe(filter((state) => state !== null))
```
Comment (proskurnicha) [inline frontend/src/app/modules/main/playlist/album-list/album-list.component.ts]: and here

**Review by proskurnicha — CHANGES_REQUESTED** [review]

**Comment by Slawisch** [comment]
Need to replace id initializer and loading in all.component.ts constructor

**Pull request merged**